test/bdd/pkg/internal/httputil/httputil.go

```diff
@@ -45,7 +46,12 @@ func DoRequest(ctx context.Context, url string, opts ...Opt) (*Response, error)
 		fn(op)
 	}
 
-	req, err := http.NewRequestWithContext(ctx, op.method, url, op.body)
+	body, err := ioutil.ReadAll(op.body)
```

Comment:
can you plz use `io` package here instead of `ioutil`

Reply:
`ioutil` allows the reading of all bytes in one call, while `io` requires creating a buffered []byte with a set size and call `Read()` until op.body is empty. 

any particular reason why `io` is more suitable in these tests?

Reply:
never mind, replace `ioutil` with `io`